workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/ActionNode.java

```diff
@@ -119,6 +119,7 @@ public void setProcessInfo(ProcessInfo processInfo) {
         } else if (processInfo.getProcessTypeId() == BASE_LOAD_ACTION) {
             BaseLoadActionNode baseLoadActionNode = new BaseLoadActionNode(this);
             containingNodes.add(baseLoadActionNode);
+        } else if (processInfo.getProcessTypeId() == ETL_ACTION) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00108)